packages/backend/src/models/OrganizationModel.ts

```diff
@@ -60,4 +60,28 @@ export class OrganizationModel {
             .returning('*');
         return OrganizationModel.mapDBObjectToOrganisation(org);
     }
+
+    async deleteOrgAndUsers(
+        organizationUuid: string,
+        userUuids: string[],
+    ): Promise<void> {
+        const [org] = await this.database(OrganizationTableName)
+            .where('organization_uuid', organizationUuid)
+            .select('*');
+        if (org === undefined) {
+            throw new NotFoundError(`No organisation found`);
+        }
+
+        await this.database.transaction(async (trx) => {
+            await trx('users').delete().whereIn('user_uuid', userUuids);
+
+            await trx('organization_memberships')
+                .delete()
+                .where('organization_id', org.organization_id);
```

Comment:
We shouldn't need this since org membership rows are deleted if the org or the user is deleted.

Reply:
mmm, I was checking all tables on postgres after deleting, and this didn't got removed. But it is true there is a foreign key.. maybe I didn't refresh the table properly, let me check again.

Reply:
Tested again, you were right. Removed